### PR TITLE
docs: implement new file structure

### PR DESCRIPTION
**Pull request by guenthermi**
docs: implement new file structure
---

This adds the file structure for our new documentation. The old doc files are stored in a separate folder and can be integrated. 

It also contains the notebook of the getting started. This however was already reviewed and will be revised later together with the rest of the documentation when it is ready.

- [ ] This PR references an open issue
- [ ] I have added a line about this change to CHANGELOG

### CODE REVIEW

**Comment by guenthermi** [comment]
netlify link which is actually working :point_right: https://ft-docs-new-file-struture--jina-docs.netlify.app/docs-new-file-struture/

File: docs/concepts/synthesis-function.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(fit-function)=
+# {octicon}`beaker` The Synthesis Function
```
Comment (LMMilliken) [inline docs/concepts/synthesis-function.md:2]: I think 'Data Synthesis' is a more informative title

**Review by scott-martens — APPROVED** [review]

File: docs/more/budget.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+```{admonition} Read full blog
+:class: hint
+Please checkout [Fine-tuning with Low Budget and High Expectations](https://jina.ai/news/fine-tuning-with-low-budget-and-high-expectations/)
+to read the full tech blog.
+```
+
+Fine-tuning takes a pre-trained model,
+trained on a related task, and then further trains it for a new task.
+Alternately, it can mean taking a model pre-trained for an open domain task, and further training it for a domain-specific one.
```
Comment (samsja) [inline docs/more/budget.md:12]: I think the definition sounds a bit weird

File: docs/Makefile [hunk 3]
```diff
@@ -20,4 +20,6 @@ help:
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
 notebook:
-	jupytext --to markdown notebooks/*.ipynb
+	mkdir -p notebooks_md
```
Comment (gmastrapas) [inline docs/Makefile:23]:
```suggestion
	mkdir -p notebooks-md
```

**Review by gmastrapas — APPROVED** [review]

File: docs/concepts/synthesis-function.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+(fit-function)=
+# {octicon}`beaker` Synthesis Training Data
```
Comment (LMMilliken) [inline docs/concepts/synthesis-function.md:2]: should be 'Synthesis of Training Data' or 'Synthesizing Training Data'

**Pull request merged**